src/Uno.UI/Extensions/ViewExtensions.wasm.cs

```diff
@@ -33,34 +32,90 @@ public static T FindFirstChild<T>(this ViewGroup view, int? childLevelLimit = nu
 		/// <param name="childLevelLimit">Defines the max depth, null if not limit (Should never be used)</param>
 		/// <param name="includeCurrent">Indicates if the current view should also be tested or not.</param>
 		/// <returns></returns>
-		public static T FindFirstChild<T>(this ViewGroup view, Func<T, bool> selector, int? childLevelLimit = null, bool includeCurrent = true)
-			where T : View
+		public static T FindFirstChild<T>(this UIElement view, Func<T, bool> selector, int? childLevelLimit = null, bool includeCurrent = true)
+			where T : UIElement
 		{
-			Func<View, bool> childSelector;
+			Func<UIElement, bool> childSelector;
 			if (selector == null)
 			{
 				childSelector = child => child is T;
 			}
 			else
 			{
-				childSelector = child =>
-				{
-					var t = child as T;
-					return t != null && selector(t);
-				};
+				childSelector = child => child is T t && selector(t);
 			}
 
-			if (includeCurrent
-				&& childSelector(view))
+			if (includeCurrent && childSelector(view))
 			{
 				return view as T;
 			}
 
-			var maxDepth = childLevelLimit.HasValue
-				? childLevelLimit.Value
-				: Int32.MaxValue;
+			var maxDepth = childLevelLimit ?? int.MaxValue;
 
 			return (T)view.EnumerateAllChildren(childSelector, maxDepth).FirstOrDefault();
 		}
+
+		public static string ShowDescendants(this UIElement view, StringBuilder sb = null, string spacing = "", UIElement viewOfInterest = null)
+		{
+			sb = sb ?? new StringBuilder();
+			AppendView(view);
+			spacing += "  ";
+			foreach (var child in view._children)
+			{
+				ShowDescendants(child, sb, spacing, viewOfInterest);
+			}
+
+			return sb.ToString();
+
+			StringBuilder AppendView(UIElement innerView)
```

Comment:
Codacy found an issue: [Change return type to 'void'; not a single caller uses the returned value.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5843545)